podman/domain/config.py

```diff
@@ -44,12 +44,12 @@ def id(self):  # pylint: disable=invalid-name
         """str: Returns identifier for service connection."""
         return self.name
 
-    @cached_property
+    @cached_property  # type: ignore[operator]
```

Comment:
For some reason, mypy thinks `cached_property` is a module and fails with a "module not callable" error.